src/components/publicApi/ChangeRecoveryScreen.tsx

```diff
@@ -12,18 +13,20 @@ interface Props {
   branding: Branding
   context: EdgeContext
   onComplete: () => void
+  onLogEvent: OnLogEvent
```

Comment:
These need to be optional otherwise we have a breaking change to the login-ui API